.github/workflows/ci.yml

```diff
@@ -84,8 +84,8 @@ jobs:
           mv dist ni-slnotebook-datasource
           zip systemlink-notebook-datasource.zip ni-slnotebook-datasource -r
 
-      - name: (Release) Run semantic-release
+      - name: (Release) Publish to Artifactory
         if: github.event_name == 'push'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: npx semantic-release
+        run: |
+          curl -H "Authorization:Bearer ${{ secrets.NIARTIFACTS_TOKEN }}" -T systemlink-notebook-datasource.zip "https://niartifacts.jfrog.io/artifactory/rnd-generic-ci/ni/systemlink/systemlink-notebook-datasource/latest.zip"
+          curl -H "Authorization:Bearer ${{ secrets.NIARTIFACTS_TOKEN }}" -T systemlink-notebook-datasource.zip "https://niartifacts.jfrog.io/artifactory/rnd-generic-ci/ni/systemlink/systemlink-notebook-datasource/${{ github.run_id }}.zip"
```

Comment:
The `github.run_id` variable is the build number. Unfortunately, it doesn't look like the build number has any significance like AzDO build numbers, and is instead a randomly generated 10-digit number. I'm wondering if we should do something to generate a more useful build number like we are [doing for Grafana](https://github.com/ni/grafana/blob/8b8f99e018b437715cc71043deae715b7d6ddc2d/.azure/azure-pipelines.yml#L17) or if we can call this sufficient.

Reply:
Alternatively we could try to grab the version generated as part of the update-versions.js script and use that instead. This might be preferable but I'm not sure what that would look like.

Reply:
Yeah, the package version would be preferable. We'd either need to parse it out of package.json or have update-version.js set an env variable. First solution might be easy actually: https://stackoverflow.com/a/18469304/6533869
It's not super important though. When we extract the plugin, Grafana will know its version, so we won't have any problems getting that info if needed.